### PR TITLE
Short URL

### LINKED ISSUE #339: Cut URL Short
image

- We do not need the https:// in front of the links, we can remove those and just show GitHub.com and rose-hack-2021.devpost.com
- Links should still be clickable, we are only showing the visible text

### PR DESCRIPTION
**Pull request by Lackline**
Shorten the URL for Table

### CODE REVIEW

File: src/components/Admin/Table.jsx [hunk 1]
```diff
@@ -14,6 +14,15 @@ import Link from "next/link";
 import { colors } from "@/data/Tags";
 import { IoIosRose } from "react-icons/io";
 import { FaCrown } from "react-icons/fa";
+import React from "react";
```
Comment (shahdivyank) [inline src/components/Admin/Table.jsx]: remove the import react line

File: src/components/Admin/Table.jsx [hunk 2]
```diff
@@ -14,6 +14,15 @@ import Link from "next/link";
...
+
+function formatLink(url) {
```
Comment (shahdivyank) [inline src/components/Admin/Table.jsx]: should be inline, remove the function

File: src/components/Admin/Table.jsx [hunk 3]
```diff
@@ -131,7 +140,7 @@ const Table = ({ headers, setHeaders, empty, setObjects, objects }) => {
                                   className="flex items-center m-0 p-0 text-black no-underline hover:!text-hackathon-blue-100 text-sm"
                                 >
                                   {icons[element.name]}
-                                  {element.link}
+                                  {formatLink(element.link)}
```
Comment (shahdivyank) [inline src/components/Admin/Table.jsx]: use replace function directly

**Comment by shahdivyank** [comment]
attach issue

**Review by shahdivyank — APPROVED** [review]

**Review by menthy-wu — APPROVED** [review]

**Pull request merged**